catalog/spatial/registry/registry-policy-plugin/src/main/java/org/codice/ddf/registry/policy/RegistryPolicyPlugin.java

```diff
@@ -108,13 +116,31 @@ public void setRegistryDisabled(boolean registryDisabled) {
         return this.registryBypassPolicyStrings;
     }
 
-    public List<String> getWriteAccessPolicyStrings() {
-        return writeAccessPolicyStrings;
+    public List<String> getCreateAccessPolicyStrings() {
+        return createAccessPolicyStrings;
     }
 
-    public void setWriteAccessPolicyStrings(List<String> writeAccessPolicyStrings) {
-        this.writeAccessPolicyStrings = writeAccessPolicyStrings;
-        parsePermissionsFromString(writeAccessPolicyStrings, writeAccessPolicy);
+    public void setCreateAccessPolicyStrings(List<String> accessPolicyStrings) {
```

Comment:
Do we care that the collections you're passing in can be modified by the caller?